src/game/character/character.gd

```diff
@@ -60,6 +69,21 @@ func getRole() -> String:
 func getTasks() -> Dictionary:
 	return _characterResource.getTasks()
 
+# set the outfit of the character
+func setAppearance(outfit: Dictionary, colors: Dictionary) -> void:
+	var outfitPaths: Dictionary = {}
+	for partGroup in outfit: ## For each customizable group
+		var selectedLook: String = outfit[partGroup]
+		for part in Appearance.groupCustomization[partGroup]: ## For each custom sprite
+			var filePath: String = Appearance.customSpritePaths[part][selectedLook]
+			outfitPaths[part] = filePath
+	
+	## Applies appearance to its skeleton
+	skeleton.applyAppearance(outfitPaths, colors)
+
+func setMainCharacter() -> void:
```

Comment:
Might wanna add a boolean input to this function, just so it's consistent with other set functions. Could even make it default to true to keep ease of use. 

There should also probably be a similar function in the character resource

Reply:
I thought the resource might not need to know if it is the main character, but maybe you're right. I'll take a look.

Reply:
So I checked, and it is the character_resource who actually uses this function just when int sets the corresponding character node.
At the moment, this variable is only used in the _ready() function of the character node, to set the camera.